tracer/build/_build/Build.ExplorationTests.cs

```diff
@@ -0,0 +1,310 @@
+                    try
+                    {
+                        SetUpExplorationTest_Debugger();
+
+                        var envVariables = GetEnvVariables(ExplorationTestUseCase.Debugger);
+                        RunExplorationTestsGitCloneAndUnitTest(envVariables);
+                        RunExplorationTestAssertions_Debugger();
+                    }
+                    finally
+                    {
+                        MoveLogsToBuildData();
+                    }
+
+                    void SetUpExplorationTest_Debugger()
+                    {
+                        Logger.Info($"Prepare environment variables for profiler.");
+                        //TODO TBD
+                    }
+
+                    void RunExplorationTestAssertions_Debugger()
+                    {
+                        Logger.Info($"Running assertions tests for debugger.");
+                        //TODO TBD
+                    }
+                })
+        ;
+
+
+    Target RunExplorationTests_ContinuousProfiler
+        => _ => _
+               .Description("Run exploration tests for profiler.")
+               .After(Clean, BuildTracerHome)
+               .Executes(() =>
+                {
+                    try
+                    {
+                        SetUpExplorationTest_ContinuousProfiler();
+                    
+                        var envVariables = GetEnvVariables(ExplorationTestUseCase.ContinuousProfiler);
+                        RunExplorationTestsGitCloneAndUnitTest(envVariables);
+                        RunExplorationTestAssertions_ContinuousProfiler();
+                    }
+                    finally
+                    {
+                        MoveLogsToBuildData();
+                    }
+
+                    void SetUpExplorationTest_ContinuousProfiler()
+                    {
+                        Logger.Info($"Prepare environment variables for profiler.");
+                        //TODO TBD
+                    }
+
+                    void RunExplorationTestAssertions_ContinuousProfiler()
+                    {
+                        Logger.Info($"Running assertions tests for profiler.");
+                        //TODO TBD
+                    }
+                })
+        ;
+
+    Dictionary<string, string> GetEnvVariables(ExplorationTestUseCase usecase)
+    {
+        var envVariables = new Dictionary<string, string>();
+        switch (usecase)
+        {
+            case ExplorationTestUseCase.Debugger:
+                envVariables.AddDebuggerEnvironmentVariables(TracerHomeDirectory);
+                break;
+            case ExplorationTestUseCase.ContinuousProfiler:
+                envVariables.AddContinuousProfilerEnvironmentVariables(TracerHomeDirectory);
+                break;
+            default:
+                throw new ArgumentOutOfRangeException(nameof(usecase), usecase, null);
+        }
+
+        envVariables.AddExtraEnvVariables(ExtraEnvVars);
```

Comment:
I think we should also default to adding the `DD_ENV` environment variable here, as it will help us separate spans created by these exploration tests. In fact, it might be nice to include `DD_VERSION` too:
```suggestion
        envVariables["DD_SERVICE"]="exploration_tests";
        envVariables["DD_VERSION"]=Version;

        envVariables.AddExtraEnvVariables(ExtraEnvVars);
```